cumulusci.yml

```diff
@@ -1,13 +1,13 @@
 project:
-    name: HEDAP
+    name: EDA
```

Comment:
I think this will break access to existing scratch orgs in the sfdx keychain.

Reply:
I'll notify the team. We can work around this via `force:alias:set`, no?

Reply:
Ah, probably yeah.